### PR TITLE
Fix current command Wordpress.md

### PR DESCRIPTION
**Pull request by renan-s-oliveira**
updated kool init wordpress to kool preset wordpress

### CODE REVIEW

File: docs/2-Presets/Wordpress.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Upon first start, if the current working directory is not an existing Wordpress
 Go to the project folder and run:
 
 ```bash
-$ kool init wordpress
+$ kool preset wordpress
 ```
 
 **kool init** creates configuration files in order to enable you to configure and extend its behaviour.
```
Comment (danielsuguimoto) [inline docs/2-Presets/Wordpress.md:20]: Please, could you change here as well?

**Review by danielsuguimoto — CHANGES_REQUESTED** [review]
@renan-s-oliveira Thanks for your contribution! I've left just one note!

File: docs/2-Presets/Wordpress.md (hunk 1 above)
Comment (renan-s-oliveira) [reply docs/2-Presets/Wordpress.md]: ok.

**Review by danielsuguimoto — APPROVED** [review]
Thx!

**Pull request merged**